src/sentry/static/sentry/app/components/organizations/headerItem.tsx

```diff
@@ -147,9 +148,15 @@ const StyledHeaderItem = styled('div', {
 `;
 
 const Content = styled('div')`
+  display: flex;
   flex: 1;
+  white-space: nowrap;
+  overflow: hidden;
   margin-right: ${space(1.5)};
-  ${overflowEllipsis};
+  div {
```

Comment:
`div` is a pretty generic selector, this can cause some issues down the line if someone were to add a component inside of `<Content>` that contains a `div`.

Instead, we should create a new styled component for this div.